SecretManagement.LastPass.psm1

```diff
@@ -0,0 +1,114 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+using namespace Microsoft.PowerShell.SecretManagement
+
+$ModuleName = 'SecretManagement.LastPass'
+
+# The last segement (underscore + number (eg: _1)) is to view how many format args are expected.
+$ErrorMessages = @{
+    GetVaultParams0                 = "At least 1 vault implementing $ModuleName must be registered."
+    GetVaultParamsMany_1            = "`$Vault argument must be provided when multiple vault implementing $ModuleName exists: {0}"
+    StayConnectedForceSwitchMissing = 'StayConnected is a sensitive operation that save the LastPass decryption key on your hard drive. If you want to proceed, reissue the command specifying the force parameter.'
+    Unregister_NotLpass_1           = "The specified vault is not a $ModuleName vault (VaultType: {0}"
+}
+
+function Connect-LastPass {
```

Comment:
Can you add basic comment-based help? In vscode (with PS ext), if you type `##` above or below the `function Connect-LastPass` line, it should scaffold it for you.

Reply:
https://github.com/TylerLeonhardt/SecretManagement.LastPass/pull/18/commits/563afb321f386e4e23f2f563f3c09d0d1bfdb4e7